src/core/components/radio/index.tsx

```diff
@@ -127,7 +127,7 @@ interface RadioProps extends InputHTMLAttributes<HTMLInputElement>, Props {
 	checked?: boolean;
 	defaultChecked?: boolean;
 	label?: ReactNode;
-	supporting?: ReactNode;
+	supporting?: ReactNode | JSX.Element;
```

Comment:
Is this intentional? I wonder what is the difference between these types, in a practical sense?

Reply:
No that's just me not paying attention. Reverted :)